sql/src/main/java/io/crate/planner/node/dql/RoutedCollectPhase.java

```diff
@@ -84,13 +85,36 @@ public RoutedCollectPhase(UUID jobId,
                               List<Projection> projections,
                               WhereClause whereClause,
                               DistributionInfo distributionInfo) {
+        this(jobId,
+            executionNodeId,
+            name,
+            routing,
+            maxRowGranularity,
+            toCollect,
+            projections,
+            whereClause,
+            distributionInfo,
+            (byte) 0);
```

Comment:
do we have that many constructor calls that it's worth creating a convenience constructor?